src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const {
+    slug: personSlug,
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+  } = person;
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+  const mother = people.find(p => p.name === motherName);
+  const father = people.find(p => p.name === fatherName);
```

Comment:
As you did previously, you can prepare people array with all information needed such as mother and father for each person